code/style.css

```diff
@@ -1,4 +1,300 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+                                                            html {
```

Comment:
Oops, some crazy indentation going on in this file 😅 Remove some indentation (should be two spaces). Reach out if you need help with VSC settings, but you should be able to go into Preferences/Settings and then search for indentation, to be able to check the Tab Size (should be 2). Make sure the box is checked that says "Insert spaces when pressing Tab" as well.

Reply:
A general comment - I guess you've been using the inspector? I use it all the time to check how the content look in all devices. You mentioned that you've only checked how it looks in mobile, but when I check e.g. iPhone X, it looks like the content is too wide (see image).

<img width="440" alt="Skärmavbild 2021-02-03 kl  17 20 09" src="https://user-images.githubusercontent.com/51529201/106776122-14573180-6644-11eb-8e2a-72f4aef81dde.png">